katas/test/src/pokerhands/joaquin/CardTest.cs

```diff
@@ -0,0 +1,53 @@
+namespace Card.Test.Joaquin;
+
+using PokerHand.Joaquin;
+
+public class CardTest
+{
+    Deck Deck = new Deck();
+
+    char[] values = { 'A', '9', '4', 'J' };
+    char[] suits = { 'C', 'D', 'H', 'S' };
+    [Fact]
+    public void ItShouldReturnAValidSuit()
+    {
+        for (int i = 0; i < values.Length; i++)
+        {
+            Card card = new Card(values[i], suits[i]);
+            Assert.Contains(card.suit, card.Deck.suits);
+        }
+    }
+
+    [Fact]
+    public void ItShouldReturnAValidCardValue()
+    {
+        for (int i = 0; i < values.Length; i++)
+        {
+            Card card = new Card(values[i], suits[i]);
+            Assert.Contains(card.value, card.Deck.cardValues);
+        }
+    }
+
+    [Fact]
+    public void ItShouldBeAValidCard()
+    {
+        for (int i = 0; i < values.Length; i++)
+        {
+            Card card = new Card(values[i], suits[i]);
+            string cardValue = card.CreateCard();
```

Comment:
This is good, but I would consider name this method as `ToString` rather than `CreateCard`. This because, `CreateCard` might give the impression that the method would return a `Card` object instead of a string.

Reply:
I'm changing it to 'CardToString' because 'ToString' name causes troubles with the implementation (it uses .ToString C# method)